### ISSUE TITLE
Web version?

### ISSUE THREAD

**Issue by manuelmeurer** [kyrylo/system_browser_client#1]
This is pretty sweet! Do you plan to offer it as a web version?
What was the reason to build it as a desktop app?

**Comment by kyrylo** [comment]
Thanks!

Currently, no plans for that. I just wanted to try something new, so I decided to go with the desktop app. I hadn't initially planned to use NW.js, but I discovered that if you want to do modern desktop apps with native look, your best bet is C++. I didn't want to write any C++.

If I chose the web version, I'd need to support JS and CSS for the wide variety browsers :) It's already a complex app, and I'd like to minimise the code and hacks.

**Comment by manuelmeurer** [comment]
Yeah I understand. When I first saw it though, I immediately thought you could offer it as a service for any Ruby repo on Github (like Coveralls/CodeClimate).
